examples/aggregate.php

```diff
@@ -28,7 +28,7 @@ function toJSON(object $document): string
 $documents = [];
 
 for ($i = 0; $i < 100; $i++) {
-    $documents[] = ['randomValue' => rand(0, 1000)];
+    $documents[] = ['randomValue' => random_int(0, 1000)];
```

Comment:
https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#randomfunctionrector

Reply:
This seems a bit overzealous, as we don't actually need a cryptographically secure number here. But it's also just an example file so I don't feel strongly about it.

Reply:
That's the new "best practice". Using it proves that the doc have been updated 😉